docs/readme-cmake.md

```diff
@@ -99,6 +99,8 @@ You can get a list of integration tests with `ctest --preset <preset>-integratio
 
 (You can also use the `<preset>-release` preset for a release build, and the output will be in the Release folder).
 
+If you want to build the game without any tests, add `--target EndlessSky` to the command.
+
```

Comment:
I don't think this is a good place. It splits two lines discussing the build preset.
It also isn't very clear as to which command it should be added to.
I think it would be more useful to describe the available build targets closer to the main block of commands above.

Something like:
````
Building all the tests alongside the game is not required.
```bash
$ cmake --build --preset <preset>-debug --target <target>
```
Without the `--target` argument, all available targets will be built.
The `EndlessSky` target will only build the game executable.
```bash
$ cmake --build --preset <preset>-debug --target EndlessSky
```
````

And, in the dependency list, further up, it would be useful to indicate that catch2 is only required for building unit tests, not the game itself.